src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/PropertyExpression.cs

```diff
@@ -38,55 +38,55 @@ private void OnPropertyChanged()
 
             if (Property.Type.IsEnumEx())
             {
-                CreateDataTypeExpressionIfNotCompatible(() => new StringExpression());
+                DataTypeExpression = new StringExpression();
```

Comment:
This caused the existing expressions to be broken (if we already manually set it). I will revert this one partially back to what it used to do (only replace if not compatible).